addon/adapters/application.js

```diff
@@ -41,7 +41,14 @@ export default Ember.Object.extend(FetchMixin, Evented, {
   */
   url: Ember.computed('type', {
     get() {
-      const config = this.container.lookupFactory('config:environment');
+      let config;
+
+      if (typeof Ember.getOwner === 'function') {
```

Comment:
Great suggestion.  I thought about it and saw it was only used a few times.  With your suggestion its probably better to pull it into a function.  I'll update the PR.